src/EoS/Nuclear/cubinterp_some.cpp

```diff
@@ -90,9 +94,9 @@ void nuc_eos_C_cubinterp_some( const real x, const real y, const real z,
 
 
 // determine location in table
-   ix = (int)( ( x - xt[0] + (real)1.0e-10 )*dxi );
-   iy = (int)( ( y - yt[0] + (real)1.0e-10 )*dyi );
-   iz = (int)( ( z - zt[0] + (real)1.0e-10 )*dzi );
+   ix = (int)floor( ( x - xt[0] )*dxi );
+   iy = (int)floor( ( y - yt[0] )*dyi );
+   iz = (int)floor( ( z - zt[0] )*dzi );
```

Comment:
- Here and other places.
  - In fact, `floor()` is not necessary as Schive corrected on slack
  - I am wondering whether removing `1.0e-10` could result in any inconsistency since it is retained in the code for table construction. Maybe you can check whether the table generated with/without `1e-10` is the same.
- Should we handle the case `i[xyz] = n[xyz] - 1`, or do you think it is not necessary? I ask this the [item](https://github.com/kuochuanpan/gamer-supernovae/issues/1#issue-924540581) is checked but no change is found here.

Reply:
Actually I found no difference between previous version with `1e-10` and this version with `floor` in any cases (driver and table constructions). But I changed this because I think this looks more reasonable and straightforward.

Reply:
- I removed `floor()`.
- `i[xyz]` this problem would (rarely/never) happen when we add `1e-10`. Since now we removed `1e-10` this won't be happened.